src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -14940,20 +14865,16 @@ void WriteLoadComponentSummaryTables(EnergyPlusData &state)
 
     // Delayed components are moved into this function so that we can calculate them one zone at a time
     // with Array1D
-    Array1D<Real64> peopleDelaySeqHeat;
-    Array1D<Real64> peopleDelaySeqCool;
```

Comment:
These arrays are temporary and used one at a time, collapsed into a single set to use for both heat and cool.